### PR TITLE
advanced topics: add ak-lite usage page

### PR DESCRIPTION
**Pull request by EmbeddedAndroid**
Basic usage guide for aktualizr-lite.

Signed-off-by: Tyler Baker <tyler@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 19c1d7530bd0160462aa93f63c1ab98af55c77ef are browesable at: https://ci.foundries.io/projects/fio-docs/builds/315/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 6927cadc744791641eea43341163ce955236f1d5 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/316/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for d7a030a20e5022f90a2e52fb8ed09d0b68ae6a44 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/317/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for dc551ab797768c204a01cf5bb32c282dc283b954 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/318/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 1f0a89b3f3f17cc932f337a55baa3e4f6832bd49 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/319/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 4ff47476fb669f41fa38a01276cbc9b45bd17d05 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/320/docs/artifacts/html/index.html

File: source/reference/aktualizr-lite.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+==============
+
+The default OTA client shipped with the Linux micrPlatform is ``aktualizr-lite``. This client is a build variant of the Aktualizr project, however is targeting users who wish to have the security aspects of TUF but do not want the complexity of Uptane.
+
+There are two modes ``aktualizr-lite`` supports.
+
+Daemon Mode (Default)
+---------------------
+
+This is the default mode of ``aktualizr-lite`` in the Linux microPlatform. It is a systemd service, which is enabled after ``lmp-device-register`` has sucessfully registered your device. The daemon will periodically check for new updates, and apply them when found.
```
Comment (doanac) [inline source/reference/aktualizr-lite.rst]: Should clarify you don't have to run lmp-device-register on the Community Factory

File: source/reference/aktualizr-lite.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+This command will apply the latest available update to the device. This includes both OSTree and Docker app targets. 
+
+``sudo aktualizr-lite update``
+
+Apply Specific Update
+~~~~~~~~~~~~~~~~~~~~~
+
+If you would like to update to a specifc build number, you can use the following command.
+
+``sudo aktualizr-lite update --update-name <build_number>``
```
Comment (doanac) [inline source/reference/aktualizr-lite.rst]: Add something like:

View Current Status
~~~~~~~~~~~~~~~~
You can run ``sudo aktualizr-lite status`` to view the current status of the device.

File: source/reference/aktualizr-lite.rst (hunk 1 above)
Comment (EmbeddedAndroid) [reply source/reference/aktualizr-lite.rst]: ah right, good point.

Comment (EmbeddedAndroid) [reply source/reference/aktualizr-lite.rst]: Updated.

File: source/reference/aktualizr-lite.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
+.. _ref-aktualizr-lite:
+
+aktualizr-lite
+==============
+
+The default OTA client shipped with the Linux micrPlatform is ``aktualizr-lite``. This client is a build variant of the Aktualizr project, however is targeting users who wish to have the security aspects of TUF but don't want the complexity of Uptane.
```
[the remark this answers is not in the document]
Comment (EmbeddedAndroid) [reply source/reference/aktualizr-lite.rst]: Done.

File: source/reference/aktualizr-lite.rst (hunk 2 above)
Comment (EmbeddedAndroid) [reply source/reference/aktualizr-lite.rst]: Added.

**Comment by doanac** [comment]
Docs for 8f4ea1aa2ee59065b2f60c935c4f24be39dcf948 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/321/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for d3c74dfaa2f226a60df4e41e2fb80aa7d9d38ccf are browesable at: https://ci.foundries.io/projects/fio-docs/builds/322/docs/artifacts/html/index.html

**Pull request merged**